src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/ConnectionSetupSubForms/PunchoutCatalogConnectionCartForm.php

```diff
@@ -0,0 +1,165 @@
+     *
+     * @return void
+     */
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'data_class' => PunchoutCatalogConnectionCartTransfer::class,
+        ]);
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addMaxDescriptionLengthField(FormBuilderInterface $builder)
+    {
+        $builder->add(PunchoutCatalogConnectionCartTransfer::MAX_DESCRIPTION_LENGTH, IntegerType::class, [
+            'label' => static::FIELD_LABEL_MAX_DESCRIPTION_LENGTH,
+            'required' => false,
+            'constraints' => [
+                new Range([
+                    'min' => 16,
+                    'max' => static::MAX_DESCRIPTION_LENGTH,
+                ]),
+            ],
+            'attr' => [
+                'template_path' => static::TEMPLATE_PATH_MAX_DESCRIPTION_LENGTH_FIELD,
+            ],
+        ]);
+
+        $builder->get(PunchoutCatalogConnectionCartTransfer::MAX_DESCRIPTION_LENGTH)
+            ->addViewTransformer($this->createMaxDescriptionLengthTransformer());
+
+        return $this;
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addEncodingField(FormBuilderInterface $builder)
+    {
+        $builder->add(PunchoutCatalogConnectionCartTransfer::ENCODING, ChoiceType::class, [
+            'label' => static::FIELD_LABEL_ENCODING,
+            'choices' => [
+                'base64' => 'base64',
+                'url-encoded' => 'url-encoded',
+                'no-encoding' => 'no-encoding',
+            ],
+            'constraints' => new NotBlank(),
+        ]);
+
+        return $this;
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addMappingField(FormBuilderInterface $builder)
+    {
+        $builder->add(PunchoutCatalogConnectionCartTransfer::MAPPING, TextareaType::class, [
+            'label' => static::FIELD_LABEL_MAPPING,
+            'required' => false,
+        ]);
+
+        return $this;
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addDefaultSupplierIdField(FormBuilderInterface $builder)
+    {
+        $builder->add(PunchoutCatalogConnectionCartTransfer::DEFAULT_SUPPLIER_ID, TextType::class, [
+            'label' => static::FIELD_LABEL_DEFAULT_SUPPLIER_ID,
+            'constraints' => [
+                new NotBlank(),
+                new Length(['max' => 64]),
+            ],
+        ]);
+
+        return $this;
+    }
+
+    /**
+     * @return \Symfony\Component\Form\CallbackTransformer
+     */
+    protected function createMaxDescriptionLengthTransformer(): CallbackTransformer
```

Comment:
`createMaxDescriptionLengthCallbackTransformer`

Reply:
Checked codebase, we do not have prefixes `CallbackTransformer` ATM,
but we have `Model` prefix for model transformers.
Renamed to createMaxDescriptionLengthViewTransformer